### PR TITLE
Update meta.description and add meta.homepage if missing

### PR DESCRIPTION
**Pull request by Pandapip1**

### CODE REVIEW

File: pkgs/bloom/default.nix [hunk 1]
```diff
@@ -35,7 +35,8 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "bloom" ];
 
   meta = {
-    description = "A release automation tool which makes releasing catkin (http://ros.org/wiki/catkin) packages easier";
+    description = "A release automation tool which makes releasing catkin packages easier";
+    fullDescription = "A release automation tool which makes releasing catkin (http://ros.org/wiki/catkin) packages easier";
```
Comment (lopsided98) [inline pkgs/bloom/default.nix:39]: `fullDescription` doesn't exist, it's `longDescription`

File: pkgs/colcon/python-setup-py.nix [hunk 2]
```diff
@@ -15,6 +15,7 @@ buildPythonPackage rec {
   doCheck = false;
 
   meta = with lib; {
+    description = "Extension for colcon-core to identify packages from setup.py files";
```
Comment (lopsided98) [inline pkgs/colcon/python-setup-py.nix:18]: Duplicate attribute, the other needs to be renamed to `longDescription`.

File: pkgs/gazebo/gz-cmake/generic.nix [hunk 3]
```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     homepage = "https://gazebosim.org/libs/cmake";
-    description = "A set of CMake modules that are used by the C++-based Gazebo projects. ";
+    description = "A set of CMake modules that are used by the C++-based Gazebo projects";
```
Comment (lopsided98) [inline pkgs/gazebo/gz-cmake/generic.nix:23]: Did you want to remove the indefinite article here like the rest?

**Review by lopsided98 — CHANGES_REQUESTED** [review]
You need to evaluate all these packages, would have caught these issues and any others that I missed.

**Comment by Pandapip1** [comment]
CI is failing on develop.

**Pull request merged**